crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -3317,13 +3317,20 @@ impl AuthorityPerEpochStore {
                             "Received RandomnessDkgMessage from {:?}",
                             authority.concise()
                         );
-                        match bcs::from_bytes(bytes) {
-                            Ok(message) => randomness_manager.add_message(authority, message)?,
-                            Err(e) => {
-                                warn!(
+                        if bytes.len() > MAX_RANDOMNESS_DKG_MESSAGE_SIZE {
```

Comment:
is this already enforced later in the flow? if not, this will require a protocol config change. even if so, it may be very tricky to avoid introducing divergent execution paths when this check is moved forward.

Reply:
this specific test was added in https://github.com/MystenLabs/sui/commit/cca0d91a5ce78ec9bb8bd2839d18ee6f2cbef745  and since we are not using this flow outside devnet, it should be safe after upgrade

Reply:
that's true (no need for backwards compat) as long as it gets in before the cut today. or gets cherrypicked in before release, i guess.